backend/src/abilities/tests/ability-fixture.ts

```diff
@@ -71,6 +88,48 @@ export function createAbilityFixture(): AbilityFixture {
         .withUpdatedAt(currentDate);
 
       return builder.build();
+    },
+    
+    // Result and error handling
+    getResult(): any {
+      return result;
+    },
+    
+    setResult(newResult: any): void {
+      result = newResult;
+    },
+    
+    getError(): Error | null {
+      return error;
+    },
+    
+    setError(newError: Error): void {
+      error = newError;
+    },
+    
+    // Then assertions
+    thenResultShouldBeArray(): void {
+      expect(Array.isArray(result)).toBe(true);
+    },
+    
+    thenResultShouldHaveLength(length: number): void {
+      expect(result).toHaveLength(length);
+    },
+    
+    thenResultShouldContainItemWithProperty(property: string, value: any): void {
+      expect(result.some((item: any) => item[property] === value)).toBe(true);
```

Comment:
Code Review: The type 'any' is used for 'item', which is too broad. Consider using a more specific type or interface to better define the structure of the item.